docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -83,6 +82,109 @@ ALTER TABLE table_identifier ADD COLUMNS ( col_spec [ , col_spec ... ] )
 </dl>
 
 
+### ALTER OR CHANGE COLUMN
+`ALTER TABLE ALTER COLUMN` or `ALTER TABLE CHANGE COLUMN` statement changes column's comment.
+
+#### Syntax
+{% highlight sql %}
+ALTER TABLE table_identifier { ALTER | CHANGE } [COLUMN] col_spec alterColumnAction
```

Comment:
super nit: ```[COLUMN]``` ->``` [ COLUMN ]```

Reply:
done

Reply:
You forgot to address the @huajianmao comment?

Reply:
done